samples/word/03-images/insert-and-get-pictures.yaml

```diff
@@ -1,6 +1,7 @@
-id: word-basics-insert-and-get-pictures
-name: Inline pictures
-description: Shows how to insert and get inline pictures.
+order: 1
+id: word-images-insert-and-get-pictures
+name: Using inline pictures
```

Comment:
Consider changing "Using" to "Use" (for consistency with how other snippet names start out)